hw/ip/soc_dbg_ctrl/data/soc_dbg_ctrl.hjson

```diff
@@ -0,0 +1,483 @@
+                    Status bit that tells whether debug authorization exchange
+                    can take place 
+                    This bit is set when auth_debug_intent_set is 1 and SoC reset
+                    sequence is paused to enable debug authorization exchange
+                  '''
+            resval: "0"
+          }
+          { bits: "5"
+            name: "auth_window_closed"
+            desc: ''' 
+                    Status bit that indicates that SoC reset sequence is
+                    unpaused SoC shall continue to boot and the debug authorization
+                    exchange cannot take place anymore until next reset. Note that
+                    the rest of the SoC reset sequence is triggered by the OT RoT
+                  '''
+            resval: "0"
+          }
+          { bits: "6"
+            name: "auth_unlock_success"
+            desc: ''' 
+                    Status bit indicating whether the unlock protocol 
+                    resulted in a successful unlock at requested level"
+                  '''
+            resval: "0"
+          }
+          { bits: "7"
+            name: "auth_unlock_failed"
+            desc: ''' 
+                    Status bit indicating whether the unlock protocol was
+                    ulted in unlock failure at requested level"
+                  '''
+            resval: "0"
+          }
+        ]
+      }
+    ]
+    jtag: [
+      { name: "JTAG_TRACE_DEBUG_POLICY_CATEGORY"
+        desc: "Trace register to observe the debug category that is either determined by hardware or software."
+        swaccess: "ro"
+        hwaccess: "hwo"
+        fields: [
+          { bits: "6:0"
+            resval: "0x0"
+            name: "category"
+            desc: '''
+                  The debug policy determined by hardware or software.
+                  '''
+          }
+        ]
+      }
+      { name: "JTAG_TRACE_DEBUG_POLICY_VALID_RELOCKED"
+        desc: "Trace register to observe the valid or relocked state that is either determined by hardware or software."
+        swaccess: "ro"
+        hwaccess: "hwo"
+        fields: [
+          { bits: "7:4"
+            mubi: true
+            resval: false
+            name: "relocked"
+            desc: '''
+                  The relocked state determined by hardware or software.
+                  '''
+          }
+          { bits: "3:0"
+            mubi: true
+            resval: false
+            name: "valid"
+            desc: '''
+                  The valid state determined by hardware or software.
+                  '''
+          }
+        ]
+      }
+      { name: "JTAG_CONTROL"
+        desc: "JTAG control register to interact with the boot flow."
+        swaccess: "rw"
+        hwaccess: "hro"
+        fields: [
+          { bits: "0"
+            resval: "0"
+            name: "boot_continue"
+            desc: '''
+                  JTAG bit to stop or continue the boot flow if Ibex.
```

Comment:
```suggestion
                  JTAG bit to stop or continue the boot flow of Ibex.
```
But more generally: What is the polarity of this bit? If set to 0, will Ibex not boot? Do we really want a reset value of 0 in that case (won't that prevent Ibex from booting until the bit gets set to 1?)?

Reply:
I added a more descriptive explanation to the FSM in the code. The halting feature only activates for TEST/RMA cases when an external halt pin is asserted. In that case, the boot continues when setting this JTAG bit. In the ordinary boot flow, i.e., when not being in RMA/TEST and no external pin is asserted, this bit does not have an influence and the boot flow continues automatically. Therefore, the polarity of 0 = halt for the default is sensible. I described the polarity also in the register description.